vpr/src/pack/attraction_groups.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * attraction_groups.h
+ *
+ *  Created on: Jun. 13, 2021
+ *      Author: khalid88
+ */
+
+#ifndef VPR_SRC_PACK_ATTRACTION_GROUPS_H_
+#define VPR_SRC_PACK_ATTRACTION_GROUPS_H_
+
+#include "vtr_strong_id.h"
+#include "vtr_vector.h"
+#include "atom_netlist.h"
+
```

Comment:
Add high-level comment. What is the purpose of attraction group, comment each member of the attraction group and say what it is used for. For example, why do I need a bool that tells me whether the region is size one